<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cv_de_modificat</class>
 <widget class="QDialog" name="cv_de_modificat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1492</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 242, 241);</string>
  </property>
  <widget class="QLabel" name="label_email_7">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Județ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Nume:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modificare">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 52, 111);
color: white;
font-size: 20px;
border-style: outset;
border-radius: 20px;
font: Times New Roman;</string>
   </property>
   <property name="text">
    <string>Modifică</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_oras">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Oraș/Localitate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_judet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_10">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Studii:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_loc">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oras">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_perioada_sfarsit_studii">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>300</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_angajator">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>210</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Număr de telefon:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Data nașterii</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Educație și calificări</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenume">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_19">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Perioada: până la</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_perioada_inceput_experienta">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>300</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Adresă de email:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_15">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Experiență profesională</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_salvare">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 52, 111);
color: white;
font-size: 20px;
border-style: outset;
border-radius: 20px;
font: Times New Roman;</string>
   </property>
   <property name="text">
    <string>Salvare</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_adresa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>461</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_16">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Funcția ocupată:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_18">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Angajator:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../PROIECT/logo_mic.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_17">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Domeniu:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_studii">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_institutie">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>471</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Instituție/Centru:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_perioada_inceput_experienta_2">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>300</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_14">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Perioada: până la</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_functia_ocupata">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nr_tel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Prenume:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_perioada_inceput_studii">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_data_nastere">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_20">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Perioada: de la</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Adresă:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_loc_studii">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Perioada: de la</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_11">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 175, 169);
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Locul:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
